src/frontend/src/lib/types/oisyDappDescription.ts

```diff
@@ -15,6 +15,10 @@ const dAppDescriptionSchema = z.object({
 	stats: z.string(),
 	logo: z.string(),
 
+	// TODO: check if text is not too long using logic from https://github.com/dfinity/portal/blob/34a0328ed4792f5a7f3943be73f13f5abaefb4b8/plugins/validate-showcase.js#L179
+	carouselSlideText: z.string().optional(),
+	carouselSlideCallToAction: z.string().optional(),
```

Comment:
Could you move these fields to oisyDappDescriptionSchema? @peterpeterparker suggested adding OISY specific fields there, as the description here matches somehow the structure of the portal.

Reply:
Done.